Modules/perfmap.c

```diff
@@ -0,0 +1,236 @@
+#include <Python.h>
+#include "frameobject.h"
+#include "internal/pystate.h"
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <sys/mman.h>
+#include <sys/syscall.h>
+#include <time.h>
+#include <unistd.h>
+
+static PyMethodDef module_methods[] = {
+    {NULL, NULL},
+};
+
+PyDoc_STRVAR(module_doc,
+"perfmap module.");
```

Comment:
where'd this file come from?

Reply:
eek, now you know what crud I have sitting about in my checkout.